static/src/javascripts/projects/common/modules/email/email-article.js

```diff
@@ -46,11 +46,15 @@ define([
                 description: 'Sign up and we\'ll send you the Guardian US Campaign Minute, once per day.',
                 successHeadline: 'Thank you for signing up to the Guardian US Campaign minute',
                 successDescription: 'We will send you the biggest political story lines of the day',
-                modClass: 'post-article',
+                modClass: config.page.isMinuteArticle ? 'post-article' : 'end-article',
                 insertMethod: function () {
-                    return function ($iframeEl) {
-                        $iframeEl.insertAfter('.js-article__container');
-                    };
+                    if (config.page.isMinuteArticle) {
+                        return function ($iframeEl) {
+                            $iframeEl.insertAfter('.js-article__container');
+                        };
+                    } else {
+                        return insertBottomOfArticle();
```

Comment:
I don't think you need to return a function from `insertBottomOfArticle` – it can just take $iframeEl as a parameter and you can get rid of the first invocation on line 150, and not call it when you return on line 56